### PR TITLE
Add authors in a separate page.

### PR DESCRIPTION
**Pull request by alarshi**
This PR addresses #5494  so that we now have authors as a separate page. Currently, the new page is under `user guide` but maybe we want to move it somewhere else (?), open to suggetions.

image

### Before your first pull request:

* [x] I have read the guidelines in our [CONTRIBUTING.md](../blob/main/CONTRIBUTING.md) document.

### For all pull requests:

* [x] I have followed the [instructions for indenting my code](../blob/main/CONTRIBUTING.md#making-aspect-better).

### CODE REVIEW

File: doc/sphinx/user/authors.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+* Wolfgang Bangerth
+* Juliane Dannberg
+* Menno Fraters
+* Rene Gassmöller
+* Anne Glerum
+* Timo Heister
+* Bob Myhill 
+* John Naliboff
+
+**with contributions by:** Jacqueline Austermann, Magali Billen, Markus B&uuml;rg, Thomas Clevenger, Samuel Cox, William Durkin, Grant Euen, Thomas Geenen, Ryan Grove, Eric Heien, Ludovic Jeanniot, Louise Kellogg, Scott King, Martin Kronbichler, Marine Lasbleis, Haoyuan Li, Shangxin Liu, Hannah Mark, Elvira Mulyukova, Bart Niday, Jonathan Perry-Houts, Elbridge Gerry Puckett, Tahiry Rajaonarison, Fred Richards, Jonathan Robey, Ian Rose, Max Rudolph, Stephanie Sparks, D. Sarah Stamps, Cedric Thieulot, Wanying Wang, Iris van Zelst, Siqi Zhang'
```
Comment (tjhei) [inline doc/sphinx/user/authors.md:15]: can you add a newline at the end? the tester complains.

Comment (tjhei) [inline doc/sphinx/user/authors.md:15]: This is the list of authors of the documentation, maybe we need to make this more obvious by adding a sentence on this page. Why are you not listed? :-)

**Review by tjhei — APPROVED** [review]

File: doc/sphinx/user/authors.md (hunk 1 above)
Comment (alarshi) [reply doc/sphinx/user/authors.md]: Not sure how we get this list. I only copied the list from `conf.py`.

File: doc/sphinx/user/authors.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+* Wolfgang Bangerth
+* Juliane Dannberg
+* Menno Fraters
+* Rene Gassmöller
+* Anne Glerum
+* Timo Heister
+* Bob Myhill
+* John Naliboff
+
+**with contributions by:** Jacqueline Austermann, Magali Billen, Markus B&uuml;rg, Thomas Clevenger, Samuel Cox, William Durkin, Grant Euen, Thomas Geenen, Ryan Grove, Eric Heien, Ludovic Jeanniot, Louise Kellogg, Scott King, Martin Kronbichler, Marine Lasbleis, Haoyuan Li, Shangxin Liu, Hannah Mark, Elvira Mulyukova, Bart Niday, Jonathan Perry-Houts, Elbridge Gerry Puckett, Tahiry Rajaonarison, Fred Richards, Jonathan Robey, Ian Rose, Max Rudolph, Stephanie Sparks, D. Sarah Stamps, Cedric Thieulot, Wanying Wang, Iris van Zelst, Siqi Zhang'
```
Comment (tjhei) [inline doc/sphinx/user/authors.md:16]:
```suggestion
**with contributions by:** Jacqueline Austermann, Magali Billen, Markus B&uuml;rg, Thomas Clevenger, Samuel Cox, William Durkin, Grant Euen, Thomas Geenen, Ryan Grove, Eric Heien, Ludovic Jeanniot, Louise Kellogg, Scott King, Martin Kronbichler, Marine Lasbleis, Haoyuan Li, Shangxin Liu, Hannah Mark, Elvira Mulyukova, Bart Niday, Jonathan Perry-Houts, Elbridge Gerry Puckett, Tahiry Rajaonarison, Fred Richards, Jonathan Robey, Ian Rose, Max Rudolph, Stephanie Sparks, D. Sarah Stamps, Cedric Thieulot, Wanying Wang, Iris van Zelst, Siqi Zhang
```

File: doc/sphinx/conf.py [hunk 3]
```diff
@@ -19,7 +19,7 @@
 
 project = 'ASPECT'
 copyright = '2023'
-author = 'Wolfgang Bangerth, Juliane Dannberg, Menno Fraters, Rene Gassmöller, Anne Glerum, Timo Heister, Bob Myhill, John Naliboff; with contributions by: Jacqueline Austermann, Magali Billen, Markus B&uuml;rg, Thomas Clevenger, Samuel Cox, William Durkin, Grant Euen, Thomas Geenen, Ryan Grove, Eric Heien, Ludovic Jeanniot, Louise Kellogg, Scott King, Martin Kronbichler, Marine Lasbleis, Haoyuan Li, Shangxin Liu, Hannah Mark, Elvira Mulyukova, Bart Niday, Jonathan Perry-Houts, Elbridge Gerry Puckett, Tahiry Rajaonarison, Fred Richards, Jonathan Robey, Ian Rose, Max Rudolph, Stephanie Sparks, D. Sarah Stamps, Cedric Thieulot, Wanying Wang, Iris van Zelst, Siqi Zhang'
+author = 'ASPECT Community'
```
Comment (bangerth) [inline doc/sphinx/conf.py:22]: The *community* is larger than just these authors. Perhaps call this "The ASPECT authors"? Or "Authors of the ASPECT manual"?

What does others think?

File: doc/sphinx/index.md [hunk 4]
```diff
@@ -3,7 +3,7 @@
 ```{admonition} Community Project
 :class: information
 
-ASPECT is a community project. Contributions to software or documentation by every user are welcome and encouraged. See [here](https://github.com/geodynamics/aspect/blob/main/CONTRIBUTING.md) for how to contribute.
+ASPECT is a community software project with the list of software developers and contributors available {ref}`sec:authors`. Contributions to software or documentation by every user are welcome and encouraged. See [here](https://github.com/geodynamics/aspect/blob/main/CONTRIBUTING.md) for how to contribute.
```
Comment (bangerth) [inline doc/sphinx/index.md:6]:
```suggestion
ASPECT is a community software project with the list of software developers and contributors available in {ref}`sec:authors`. Contributions to software or documentation by every user are welcome and encouraged. See [here](https://github.com/geodynamics/aspect/blob/main/CONTRIBUTING.md) for how to contribute.
```

File: doc/sphinx/user/authors.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
+(sec:authors)=
+# ASPECT community
```
Comment (bangerth) [inline doc/sphinx/user/authors.md:2]: Same here in the section title.

File: doc/sphinx/user/authors.md (hunk 2 above)
Comment (bangerth) [inline doc/sphinx/user/authors.md:16]:
```suggestion
**With contributions by:** Jacqueline Austermann, Magali Billen, Markus B&uuml;rg, Thomas Clevenger, Samuel Cox, William Durkin, Grant Euen, Thomas Geenen, Ryan Grove, Eric Heien, Ludovic Jeanniot, Louise Kellogg, Scott King, Martin Kronbichler, Marine Lasbleis, Haoyuan Li, Shangxin Liu, Hannah Mark, Elvira Mulyukova, Bart Niday, Jonathan Perry-Houts, Elbridge Gerry Puckett, Tahiry Rajaonarison, Fred Richards, Jonathan Robey, Ian Rose, Max Rudolph, Stephanie Sparks, D. Sarah Stamps, Cedric Thieulot, Wanying Wang, Iris van Zelst, Siqi Zhang'
```

**Review by bangerth — CHANGES_REQUESTED** [review]

File: doc/sphinx/conf.py (hunk 3 above)
Comment (alarshi) [reply doc/sphinx/conf.py:22]: I like "Authors of the ASPECT manual". I will change it to this. Thank you!

**Comment by alarshi** [comment]
Thank you for the changes! I addressed them in the latest commit.

File: doc/sphinx/user/index.md [hunk 6]
```diff
@@ -4,6 +4,7 @@
 ---
 maxdepth: 2
 ---
+authors.md
```
Comment (gassmoeller) [inline doc/sphinx/user/index.md:7]: I would prefer if this is not the first page in the documentation, it seems a bit excessive (also we already link to it from the start page). Can we put it last, after `answers.md`?

**Review by gassmoeller — COMMENTED** [review]
Very nice! What do you think about my suggestions?

File: doc/sphinx/index.md [hunk 7]
```diff
@@ -3,7 +3,7 @@
 ```{admonition} Community Project
 :class: information
 
-ASPECT is a community project. Contributions to software or documentation by every user are welcome and encouraged. See [here](https://github.com/geodynamics/aspect/blob/main/CONTRIBUTING.md) for how to contribute.
+ASPECT is a community software project with the list of software developers and contributors available in {ref}`sec:authors`. Contributions to software or documentation by every user are welcome and encouraged. See [here](https://github.com/geodynamics/aspect/blob/main/CONTRIBUTING.md) for how to contribute.
```
[the remark this answers is not in the document]
Comment (alarshi) [reply doc/sphinx/index.md:6]: This works! Thanks :)

**Comment by alarshi** [comment]
Thank you for the suggestions! I agree that it looks neater now:

image

**Review by gassmoeller — APPROVED** [review]
Nice, thank you!

**Review by bangerth — APPROVED** [review]

**Pull request merged**